frontend/pages/legal/acceptable-use-policy.html

```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Acceptable Use Policy</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="acceptable use policy" />
+    <meta name="author" content="" />
+
+    <!-- Le styles -->
+    <link href="/css/bootstrap.css" rel="stylesheet" />
+    <link href="/css/bootstrap-responsive.css" rel="stylesheet" />
+    <link href="/css/style.css" rel="stylesheet" />
+    <link href="/css/datepicker.css" rel="stylesheet" />
+    <link href="/css/jquery-ui-1.8.16.custom.css" rel="stylesheet" />
+    <link href="/css/jquery-ui-timepicker-addon.css" rel="stylesheet" />
```

Comment:
React applications do not use jquery plugins

Reply:
removed